name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/KeytradeBankPDFExtractor.java

```diff
@@ -211,9 +216,18 @@ private void addDividendeTransaction()
     {
         transaction
                 // Verrechnungssteuer 26,38 % 2,03 EUR
+                .section("tax", "currency").optional()
+                .match("^Verrechnungssteuer [\\.,\\d]+ % (?<tax>[\\.,\\d]+) (?<currency>[\\w]{3})$")
+                .assign((t, v) -> processTaxEntries(t, v, type))
+
                 // Impôt à la source 26,38 % 16,35 EUR
                 .section("tax", "currency").optional()
-                .match("^(Verrechnungssteuer|Imp.t . la source) [\\.,\\d]+ % (?<tax>[\\.,\\d]+) (?<currency>[\\w]{3})$")
+                .match("^Imp.t . la source [\\.,\\d]+ % (?<tax>[\\.,\\d]+) (?<currency>[\\w]{3})$")
+                .assign((t, v) -> processTaxEntries(t, v, type))
+
+                // Taxe boursière - 0,35% - 5,22 EUR
```

Comment:
There is no "Taxe boursière" and "courtage" with the dividend (at least for the Belgian document).
It is replaced by "Précompte mobilier belge" like you can see [here](https://forum.portfolio-performance.info/t/pdf-import-von-keytrade-bank/15416/27?u=scrayn).